include/kodlab_mjbots_sdk/pid_controller.h

```diff
@@ -0,0 +1,648 @@
+/**
+ * @file pid.h
+ * @author Chandravaran Kunjeti (kunjeti@seas.upenn.edu)
+ * @author Raghavesh Viswanath (vrag@seas.upenn.edu)
+ * @brief Class implementation of PID
+ * @date 5/18/2023
+ * @copyright 2023 The Trustees of the University of Pennsylvania.
+ * All rights reserved.
+ *
+ */
+
+#pragma once
+
+#include <math.h>
+#include <array>
+#include <vector>
+#include <algorithm>
+#include <Eigen/Dense>
+#include <tuple>
+
+namespace kodlab {
+
+/**
+ * @brief PIDController class that allows you to use PID control.
+ */
```

Comment:
Your going to need to add a comment somewhere that the accumulator limit is a limit on integral error, not on ki * integral error. I would have expected it to be the later. @jdcaporale

Reply:
That makes sense to me, essentially limiting the effort that is allowed by the accumulator. That is more intuitive so let's do that.

Reply:
Still needs to be done